src/views/Factboxes/index.tsx

```diff
@@ -89,22 +98,27 @@ const FactboxItem = ({ factbox, openFactbox, locale, timeZone }: FBItem): JSX.El
       <div className='w-3/4 flex flex-col' draggable>
         <p className='w-full truncate text-sm font-bold'>{title}</p>
         <p className='w-full truncate text-xs'>{text}</p>
+        <p className='w-full truncate text-xs'>
+          {/* eslint-disable-next-line */}
+          <em>Skapad {createdFormatted}</em>
+        </p>
         {modified
           ? (
               <p className='w-full truncate text-xs'>
                 <em>
-                  Senast ändrad
-                  {convertedDate}
+                  {/* eslint-disable-next-line */}
+                  Senast ändrad {convertedDate}
```

Comment:
It will otherwise give an error:
```
{convertedDate}` must be placed on a new lineeslint[@stylistic/jsx-one-expression-per-line](https://eslint.style/rules/jsx/jsx-one-expression-per-line)
```

The reason why I looked at it from the start was because the label said:
```
Senast ändradden 16 nov...
```
, so they were mashed together. I admit to preferring this way of writing instead of the below 😄 

 ```
<em>
  Senast ändrad <-- space
  {convertedDate}
</em>
```

But it's really not an issue. Either way is fine, I just chose to include it.